rpc/schemas/common/devices/DeviceRegistration.json

```diff
@@ -31,7 +31,8 @@
         "__type": true,
         "deviceId": { "type": "string" },
         "deviceDisplayName": { "type": [ "string", "null" ] },
-        "registrationCreatedOn": { "type": "string", "format": "date-time" }
+        "registrationCreatedOn": { "type": "string", "format": "date-time" },
+        "additionalSpecifications": { "type": [ "object", "string", "null" ] }
```

Comment:
This can reference `ApplicationData.json` for consistency with other places where this is used.